bin.src/genDefectFits.py

```diff
@@ -16,6 +17,9 @@ def genDefectFits(source, targetDir):
     mapper = PfsMapper(root=".")
     camera = mapper.camera
 
+    logger = log.Log.getLogger("genDefectFits")
+    logger.setLevel(log.INFO)
```

Comment:
Why are we over-riding the default log level by default?

The argument parser has a `--verbose` flag. Why *aren't* we setting the log level based on that?